tmva/tmva/inc/TMVA/DNN/Architectures/Reference.h

```diff
@@ -317,7 +317,7 @@ class TReference
 
    /** @name Regularization
     * For each regularization type two functions are required, one named
-    * <tt><Type>Regularization</tt> that evaluates the corresponding
+    * <tt>`<Type>`Regularization</tt> that evaluates the corresponding
```

Comment:
Same as above

Reply:
This leads to a warning when running doxygen

Reply:
```
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/DNN/Architectures/Cuda.h:419: warning: Unsupported xml/html tag <Type> found
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/DNN/Architectures/Reference.h:337: warning: Unsupported xml/html tag <Type> found
```

Reply:
Simplified.